tests/core/kernel-and-base-single-reboot-failover/task.yaml

```diff
@@ -0,0 +1,117 @@
+summary: Exercises a simultaneous kernel and base refresh with a single reboot
+
+# TODO make the test work with ubuntu-core-20
+systems: [ubuntu-core-18-*]
+
+environment:
+    BLOB_DIR: $(pwd)/fake-store-blobdir
+
+prepare: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+    snap ack "$TESTSLIB/assertions/testrootorg-store.account-key"
+    #shellcheck source=tests/lib/store.sh
+    . "$TESTSLIB"/store.sh
+
+    setup_fake_store "$BLOB_DIR"
+
+    readlink /snap/pc-kernel/current > pc-kernel.rev
+    readlink "/snap/core18/current" > core.rev
```

Comment:
nitpick^1000: it looks a bit weird that one line has the path quoted, and the other one doesn't.